servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -424,4 +425,14 @@ public void userHeartbeat(long userId, List<Long> metrics) {
       mWorkerSource.incBytesWrittenUfs(metrics.get(Constants.BYTES_WRITTEN_UFS_INDEX));
     }
   }
+
+  /**
+   * Set the pinlist for the underlying blockstore.
+   * Typically called by PinListSyncer.
+   *
+   * @param list
+   */
+  public void setPinList(Set<Integer> list) {
```

Comment:
* This javadoc is not clear what are the integers in the list.
* describe parameter `list` in javadoc